05.host/01.your-domain/02.configure-settings/docs.md

```diff
@@ -8,8 +8,9 @@ While you can use the sandbox as is, we recommend that you configure your domain
 **On This Page:**
 
 + [Configure Basic Server Settings](#configure-basic-server-settings)
-	+ [Setup Wizard](#setup-wizard)
-	+ [Domain Server Settings](#domain-server-settings)
+  + [Setup Wizard](#setup-wizard)
```

Comment:
Fixed this. This is definitely an issue with the markdown editor I use. I use one called Typora because I like seeing how it will look before testing it locally as well.